README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
```

Comment:
Before we get too lost in the weeds, the overriding point is that the best place to implement this is in the server to ensure consistent behavior for all, and to ensure that the veneer doesn't shadow/alter server behavior.